android/src/main/java/io/dcvz/rnzendesk/RNZendeskBridge.java

```diff
@@ -247,12 +249,23 @@ public void getRequests(final String statuses, final Promise promise) {
             @Override
             public void onSuccess(List<Request> requests) {
                 for (Request r : requests) {
+                    List<User> commentingAgents = r.getLastCommentingAgents();
+                    WritableArray commentingAgentAvatarUrls = new WritableNativeArray();
+                    for (User user : commentingAgents) {
+                        Attachment avatar = user.getPhoto();
+
+                        if (avatar != null) {
+                            commentingAgentAvatarUrls.pushString(avatar.getContentUrl());
+                        }
+                    }
+
                     WritableMap request = new WritableNativeMap();
                     request.putString("id", r.getId());
                     request.putString("status", r.getStatus().name().toLowerCase());
                     request.putString("subject", r.getSubject());
                     request.putString("lastComment", r.getLastComment().getBody());
                     request.putString("updatedAt", Long.toString(r.getUpdatedAt().getTime()));
+                    request.putArray("avatarUrls", commentingAgentAvatarUrls);
```

Comment:
think it might be good to send back the user IDs as well as the avatarUrls - even if we can get the id from the url (can we do that?)

We don't really **need** that though so up to you

Reply:
Yeah we don't need it and we can't do anything with it (there's nothing to help us get more information about users). Think this will be a good addition if/when we need.